src/adapters/api-blueprint-adapter.coffee

```diff
@@ -298,9 +297,9 @@ legacyResourcesFrom1AResource = (legacyUrlConverterFn, resource, sourcemap, opti
       legacyResource.model = {}
 
     legacyResource.resourceParameters = resourceParameters or []
-    legacyResource.actionParameters   = actionParameters or []
+    legacyResource.actionParameters   = getParametersOf(action, options) or []
 
-    legacyResource.parameters = actionParameters or resourceParameters or []
+    legacyResource.parameters = legacyResource.resourceParameters.concat(legacyResource.actionParameters)
```

Comment:
So far that change is a breaking non-backwards compatible change.

The `concat` was not done in App-AST, because it'll be impossible to distinguish between those groups of param sources.